cityofaustin/atd-moped
# Fix duplicating project type bug

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9070

## Testing
**URL to test:** 

needs to test on test. I have again branched from another PR, so you could in theory only test on this branch and see the status update and funding source column.

**Steps to test:**

Add and remove multiple project types to a project. Go back to list view and the project should not be duplicated. Basically you dont want to see this:
<img width="1003" alt="Screen Shot 2022-05-04 at 4 39 08 PM" src="https://user-images.githubusercontent.com/12474808/166830152-005bd16e-4eb3-40e5-97a6-58dc621d9005.png">



---
#### Ship list
